Dockerfile

```diff
@@ -1,51 +1,13 @@
-FROM debian:jessie
-MAINTAINER EverydayHero <edh-dev@everydayhero.com.au>
+FROM quay.io/everydayhero/base:0.1
+MAINTAINER Everydayhero Engineering "edh-dev@everydayhero.com.au"
 
-RUN apt-key adv --keyserver pgp.mit.edu --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62
-RUN echo "deb-src http://nginx.org/packages/mainline/debian/ jessie nginx" >> /etc/apt/sources.list
+ADD build.sh /tmp
+RUN /tmp/build.sh
 
-ENV NGINX_VERSION 1.9.4-1~jessie
-ENV NGINX_SOURCE_FOLDER nginx-1.9.4
+RUN ln -sf /dev/stdout /var/log/nginx/access.log
+RUN ln -sf /dev/stderr /var/log/nginx/error.log
```

Comment:
Why not `RUN ln -sf /dev/stdout /var/log/nginx/access.log && ln -sf /dev/stderr /var/log/nginx/error.log`? Makes it one less layer.